scripts/bump-cdn-version.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+/**
+ * @fileoverview
+ * Updates the CDN URLs in the README.md to the latest major version. Does
+ * nothing if the README.md is already up-to-date.
+ *
+ * It assumes the major version in the README is either equal to the current
+ * package major version, or the package minor version minus 1. If not the
+ * behaviour of this script should be consider undefined.
+ */
+
+const fs = require("fs");
+const path = require("path");
+const getMajorVersion = require("semver/functions/major");
+
+const rootDir = path.resolve(__dirname, "..");
+const PACKAGE_JSON_FILE = path.resolve(rootDir, "package.json");
+const README_FILE = path.resolve(rootDir, "README.md");
```

Comment:
The style of these variable names should be consistent with the other scripts, basically the same suggestion of https://github.com/simple-icons/simple-icons/pull/5546#discussion_r619867415

 ```suggestion
const packageJsonFile = path.resolve(rootDir, "package.json");
const readmeFile = path.resolve(rootDir, "README.md");
```